Readme.md

```diff
@@ -63,13 +66,51 @@ As I explained in a [blog post](http://www.matez.de/index.php/2020/06/22/the-fut
     * In order to use curve25519-sha256, curve448-sha512 & chacha20-poly1305@<!-- -->openssh.com, you must use at least Java 11.
 
 ## Changes since fork:
+* [0.1.66](https://github.com/mwiede/jsch/releases/tag/jsch-0.1.66)
+  * Added support for [RFC 8308](https://datatracker.ietf.org/doc/html/rfc8308) extension negotiation and server-sig-algs extension
+    * This support is enabled by default, but can be controlled via the enable_server_sig_algs config option (or `jsch.enable_server_sig_algs` system property)
+    * When enabled and a server-sig-algs message is received from the server, the algorithms included by the server and also present in the PubkeyAcceptedKeyTypes config option will be attempted first when using publickey authentication
+    * Additionally if the server is detected as OpenSSH version 7.4, the rsa-sha2-256 & rsa-sha2-512 algorithms will be added to the received server-sig-algs as a workaround for [OpenSSH bug 2680](https://bugzilla.mindrot.org/show_bug.cgi?id=2680)
+  * Added support for various algorithms supported by Tectia (ssh.com):
+    * diffie-hellman-group14-sha224@<!-- -->ssh.com
+    * diffie-hellman-group14-sha256@<!-- -->ssh.com
+    * diffie-hellman-group15-sha256@<!-- -->ssh.com
+    * diffie-hellman-group15-sha384@<!-- -->ssh.com
+    * diffie-hellman-group16-sha384@<!-- -->ssh.com
+    * diffie-hellman-group16-sha512@<!-- -->ssh.com
+    * diffie-hellman-group18-sha512@<!-- -->ssh.com
+    * diffie-hellman-group-exchange-sha224@<!-- -->ssh.com
+    * diffie-hellman-group-exchange-sha384@<!-- -->ssh.com
+    * diffie-hellman-group-exchange-sha512@<!-- -->ssh.com
+    * hmac-sha224@<!-- -->ssh.com
+    * hmac-sha256@<!-- -->ssh.com
+    * hmac-sha256-2@<!-- -->ssh.com
+    * hmac-sha384@<!-- -->ssh.com
+    * hmac-sha512@<!-- -->ssh.com
+    * ssh-rsa-sha224@<!-- -->ssh.com
+    * ssh-rsa-sha256@<!-- -->ssh.com
+    * ssh-rsa-sha384@<!-- -->ssh.com
+    * ssh-rsa-sha512@<!-- -->ssh.com
+  * Added support for SHA224 to FingerprintHash
+  * Fixing [#52](https://github.com/mwiede/jsch/issues/52)
+  * Deprecate `void setFilenameEncoding(String encoding)` in favor of `void setFilenameEncoding(Charset encoding)` in `ChannelSftp`
+  * Added support for rsa-sha2-256 & rsa-rsa2-512 algorithms to `ChannelAgentForwarding`
+  * Address [#65](https://github.com/mwiede/jsch/issues/65) by adding ssh-agent support derived from [jsch-agent-proxy](https://github.com/ymnk/jsch-agent-proxy)
+    * See `examples/JSchWithAgentProxy.java` for simple example
+    * ssh-agent support requires either [Java 16's JEP 380](https://openjdk.java.net/jeps/380) or the addition of [junixsocket](https://github.com/kohlschutter/junixsocket) to classpath
+    * Pageant support is untested & requires the addition of [JNA](https://github.com/java-native-access/jna) to classpath
```

Comment:
@norrisjeremy I think we should put release notes at the releases in the futures. Otherwise one gets lost on the readme.

Reply:
Do you mean splitting them out into a ChangeLog.md?